3.processing_features/scripts/0.merge_sc_cytotable.py

```diff
@@ -116,7 +116,7 @@
 # In[5]:
 
 
-converted_df = pd.read_parquet(plate_info_dictionary["Plate_5"]["dest_path"])
+converted_df = pd.read_parquet(plate_info_dictionary["Plate_4"]["dest_path"])
```

Comment:
Are you just sampling plate 4 to assess if all of the other plates look correct?

Reply:
Its just a check to make sure that the number of single-cells matches that of the SQLite file and can be updated to load in multiple. Definitely a future idea!